moskito-webui/src/main/java/net/anotheria/moskito/webui/auth/actions/LoginAction.java

```diff
@@ -0,0 +1,33 @@
+package net.anotheria.moskito.webui.auth.actions;
+
+import net.anotheria.maf.action.Action;
+import net.anotheria.maf.action.ActionCommand;
+import net.anotheria.maf.action.ActionMapping;
+import net.anotheria.maf.bean.FormBean;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginAction implements Action{
+
+    @Override
+    public void preProcess(ActionMapping mapping, HttpServletRequest req, HttpServletResponse res) throws Exception {
+
+    }
+
+    @Override
+    public ActionCommand execute(ActionMapping mapping, FormBean formBean, HttpServletRequest req, HttpServletResponse res) throws Exception {
+        req.setAttribute("title", "Log in Moskito-Inspect");
+
+        if("1".equals(req.getParameter("error")))
```

Comment:
getParameter(String) method returns string or null. Calling equals() on getParameter() result may produce null pointer exception. This expression is still compact and never produce null pointer exception